v2/dtos/responses/deviceservice.go

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package responses
+
+import (
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos/common"
+)
+
+// AddDeviceServiceResponse defines the Response Content for POST DeviceService DTOs.
+// This object and its properties correspond to the AddDeviceServiceResponse object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/AddDeviceServiceResponse
+type AddDeviceServiceResponse struct {
+	common.BaseResponse `json:",inline"`
+	Id                  string `json:"id"`
+}
+
+// UpdateDeviceServiceResponse defines the Response Content for PUT DeviceService as pushed DTO.
+// This object and its properties correspond to the UpdateDeviceServiceResponse object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/UpdateDeviceServiceResponse
+type UpdateDeviceServiceResponse struct {
+	common.BaseResponse `json:",inline"`
+	Id                  string `json:"id"`
+}
+
+// DeviceServiceResponse defines the Response Content for GET DeviceService DTOs.
+// This object and its properties correspond to the DeviceServiceResponse object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/DeviceServiceResponse
+type DeviceServiceResponse struct {
```

Comment:
Doesn't match the Swagger:
```
    DeviceServiceResponse:
      allOf:
        - $ref: '#/components/schemas/BaseResponse'
      type: object
      properties:
        service:
          $ref: '#/components/schemas/DeviceService'
```

Reply:
I will update the schema from the Swagger file.